site/docs/components/avatar/accessibility.mdx

```diff
@@ -9,6 +9,6 @@ data:
 
 ### Best practices
 
-Decorative Avatars (Avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an Avatar for decorative purposes, pass `aria-hidden=”true”` to hide its label from the screen reader.
+Decorative Avatars (Avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an Avatar for decorative purposes, pass `aria-hidden={true}` to hide its label from the screen reader.
```

Comment:
aria-hidden is Booleanish, but doesn't hurt to change it to true

Reply:
I'm tempted to say this whole sentence should be something like:

```suggestion
Decorative Avatars (Avatars that repeat the information already expressed by the accompanying text) should be hidden from the screen reader to avoid repetition. If you are adding an Avatar for decorative purposes, use `aria-hidden` to hide its label from the screen reader.
```

Reply:
cc @jpmorganchase/salt-content

Reply:
It's actually not so simple, I'll chat with you

Reply:
agreed, will change to Joshs suggestion if its ok? @mikearildbrown